sgbackend/mail.py

```diff
@@ -65,7 +65,13 @@ def send_messages(self, emails):
 
     def _build_sg_mail(self, email):
         mail = sendgrid.Mail()
-        mail.add_to(email.to)
+        
+        # use smtpapi if recipient list provided, otherwise use standard to field
+        if len(email.to) > 1:
```

Comment:
Looks like any string with more than 1 char will do here and the "tos" will be added letter by letter.
Better check for isinstance?